shared/src/business/entities/cases/Case.ts

```diff
@@ -1550,10 +1550,27 @@ export class Case extends JoiValidationEntity {
    * @returns {Case} the updated case entity
    */
   setCaseCaption(caseCaption) {
+    
     this.caseCaption = caseCaption;
+    
+    // New method to find case in case array and update that caption as well
```

Comment:
We love comments that give additional context or explain things that aren't readily apparent from otherwise self-descriptive code. This comment, however, is unnecessary.